### PR TITLE
Updated entrypoint script to accept output directory as an argument

### PR DESCRIPTION
**Pull request by kichalla**

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -1,46 +1,72 @@
...
 echo
-oryxCommand="oryx build ${sourceDirectory}"
+oryxCommand="oryx build $sourceDirectory"
 
-echo
+if [ ! -z "$outputDirectory" ]
+then
+    outputDirectory=$(getAbsolutePath "$outputDirectory")
+    oryxCommand="$oryxCommand -o $outputDirectory"
+fi
```
Comment (qianz2) [inline entrypoint.sh]: We could also print some outputs for this one: `echo "No output directory was provided"`
 and `echo "Output directory was provided -- the following directory will be built: '${outputDirectory}'"`

File: entrypoint.sh [hunk 2]
```diff
@@ -53,6 +79,6 @@ if [ -z "$ORYX_DISABLE_TELEMETRY" ] || [ "$ORYX_DISABLE_TELEMETRY" == "false" ];
     export GITHUB_ACTIONS_BUILD_IMAGE_PULL_END_TIME=$endTime
 fi
 
-export ORYX_SDK_STORAGE_BASE_URL="https://oryxsdk-cdn.azureedge.net"
```
Comment (qianz2) [inline entrypoint.sh:56]: We don't want this anymore?

Comment (kichalla) [reply entrypoint.sh:56]: Oops...that was a mistake. Good catch.

File: entrypoint.sh (hunk 1 above)
Comment (kichalla) [reply entrypoint.sh]: Sure

File: action.yml [hunk 3]
```diff
@@ -10,6 +10,9 @@ inputs:
   platform-version:
     description: 'Version of the programming platform used to build the web app; if no value is provided, Oryx will determine the version needed to build the repository.'
     required: false
+  output-directory:
+    description: 'The directory where the build output has to be put.'
+    required: false
```
Comment (arroyc) [inline action.yml:15]: does this mean optional input arg?

File: entrypoint.sh [hunk 4]
```diff
@@ -1,46 +1,75 @@
...
+    oryxCommand="$oryxCommand --platform $platform"
+fi
+
+echo
+if [ -z "$platformVersion" ]
+then
     echo "No platform version provided -- Oryx will determine the version."
+else
+    echo "Platform version provided: '$platformVersion'"
+    oryxCommand="$oryxCommand --platform-version $platformVersion"
```
Comment (arroyc) [inline entrypoint.sh:66]: nit:  just echo the oryx command out of if/else

File: entrypoint.sh [hunk 5]
```diff
@@ -1,46 +1,75 @@
...
-if [ -n "${platform}" ]
+if [ -z "$outputDirectory" ]
 then
-    echo "Platform provided: '${platform}'"
-    oryxCommand="${oryxCommand} --platform ${platform}"
+    echo "No output directory was provided"
 else
-    echo "No platform provided -- Oryx will enumerate the source directory to determine the platform."
+    outputDirectory=$(getAbsolutePath "$outputDirectory")
+    oryxCommand="$oryxCommand -o $outputDirectory"
```
Comment (arroyc) [inline entrypoint.sh:49]: nit: just declare the oryxcommand as a variable may be?

File: entrypoint.sh [hunk 6]
```diff
@@ -1,46 +1,75 @@
...
+    (*) echo ""
+    esac
+}
 
-echo
+getAbsolutePath(){
+    local path="$1"
+
+    if [ -z "$path" ]; then
+        echo "$PWD"
```
Comment (arroyc) [inline entrypoint.sh:16]: nit: add additional echo that $path is empty

File: action.yml (hunk 3 above)
Comment (kichalla) [reply action.yml:15]: yes

File: entrypoint.sh (hunk 6 above)
Comment (kichalla) [reply entrypoint.sh:16]: We have that at the call site

File: entrypoint.sh (hunk 5 above)
Comment (kichalla) [reply entrypoint.sh:49]: Can you clarify?...`oryxCommand` is already a variable

File: entrypoint.sh (hunk 4 above)
Comment (kichalla) [reply entrypoint.sh:66]: Oryx command is being echoed on line 88

**Review by qianz2 — APPROVED** [review]

**Pull request merged**